templates/bastian.yaml

```diff
@@ -10,7 +10,7 @@ Parameters:
   InstanceType:
     Description: WebServer EC2 instance type
     Type: String
-    Default: t3.nano
+    Default: t3.micro
```

Comment:
this will apply to every environment (dev, stage, and prod) once merged.

Reply:
That was the intent, as we need to be able to deploy data to all of the environments.